packages/@okta/vuepress-site/docs/guides/enable-express-configuration/main/SCIM/assign-client-grant.md

```diff
@@ -0,0 +1,27 @@
+Run the following commands to create the Client Credentials. Ensure that you update the `client_id` value with the OIDC app client ID and include the Auth0 domain in the `audience` parameter.
```

Comment:
This is incorrect. This step is to update client-grants. Could you check the draft again?
https://github.com/atko-scratch/haisheng-scratch/blob/main/express-configure/auth0-tenant-setup-docs/ISV-instruction-SCIM.org#grant-scim-scopes-for-the-oin-client